modules/storages/app/components/storages/admin/forms/oauth_client_form_component.html.erb

```diff
@@ -60,7 +60,14 @@
                 data: {
                   'storages--oauth-client-form-target': 'redirectUri'
                 })
-            end
+            end)
+
+            # NOTE: Hardcoded `FormControl-caption` class is used to align the text with the input field.
+            #      This is a workaround until Primer::OpenProject::InputGroup supports a caption.
+            #      See https://community.openproject.org/wp/51376
+            concat(render(Primer::Beta::Text.new(classes: 'FormControl-caption')) do
```

Comment:
@HDinger I created https://community.openproject.org/wp/51376 to cover this issue